src/patterns/data/default/vein_and_venule_pattern.tsv

```diff
@@ -37,3 +37,13 @@ UBERON:8600105	lumbar vein	FMA:70889	UBERON:0001638	UBERON:0002355	A vein that i
+UBERON:8920012	segmental renal vein		UBERON:0001638	UBERON:0002113	A vein that is a tributary of the renal vein and drains blood from a segment of the kidney.	https://www.ncbi.nlm.nih.gov/books/NBK538298/|doi:10.37532/1308-4038.18.11.26				UBERON:0001140			https://orcid.org/0000-0001-6757-4744	14/04/2025 13:00:33
+UBERON:8920013	anterior ciliary vein	FMA:51798	UBERON:0001638	UBERON:0000970	A vein that is a tributary of the superior opthalmic vein and drains blood from the ciliary body of the eye.	https://www.kenhub.com/en/library/anatomy/ophthalmic-veins|https://www.imaios.com/en/e-Anatomy/Anatomical-Parts/Anterior-ciliary-veins|https://www.nature.com/articles/s41433-023-02416-z				UBERON:0011192		UBERON:0001775	https://orcid.org/0000-0001-6757-4744	14/04/2025 13:00:33
+UBERON:8920014	left bronchial vein	FMA:4837	UBERON:0001592	UBERON:0002048	A bronchial vein that drains blood from the bronchi and hilar structures into the accessory hemiazygos vein or the left superior intercostal vein.	https://www.elsevier.com/resources/anatomy/cardiovascular-system/veins/bronchial-veins-right/21896|wikipedia:Bronchial_veins				UBERON:0006665|UBERON:0012195		UBERON:0002185|UBERON:0004886	https://orcid.org/0000-0001-6757-4744	14/04/2025 13:00:33
+UBERON:8920015	right bronchial vein	FMA:4942	UBERON:0001592	UBERON:0002048	A bronchial vein that drains blood from the bronchi and hilar structures into the azygos vein.	https://www.elsevier.com/resources/anatomy/cardiovascular-system/veins/bronchial-veins-right/21896|wikipedia:Bronchial_veins				UBERON:0001594		UBERON:0002185|UBERON:0004886	https://orcid.org/0000-0001-6757-4744	14/04/2025 13:00:33
+UBERON:8920016	right inferior pulmonary vein	FMA:49911	UBERON:0002016	UBERON:0002048	A pulmonary vein that drains blood from the inferior lobe of the right lung.	https://www.elsevier.com/resources/anatomy/cardiovascular-system/veins/right-inferior-pulmonary-vein/20294|wikipedia:Pulmonary_vein	Right inferior pulmonary venous tree|Right lower lobar vein	FMA:49911				UBERON:0002171	https://orcid.org/0000-0001-6757-4744	14/04/2025 13:00:33
+UBERON:8920017	left inferior pulmonary vein	FMA:49913	UBERON:0002016	UBERON:0002048	A pulmonary vein that drains blood from the inferior lobe of the left lung.	https://www.elsevier.com/resources/anatomy/cardiovascular-system/veins/left-inferior-pulmonary-vein/20251|wikipedia:Pulmonary_vein	Left inferior pulmonary venous tree|Left lower lobar vein	FMA:49913				UBERON:0008953	https://orcid.org/0000-0001-6757-4744	14/04/2025 13:00:33

```

Comment:
I would drop the elsevier anatomy link. The wikipedia and FMA links should suffice.